docs/process.md

```diff
@@ -173,12 +173,12 @@ Their full descriptions can be viewed under the [labels page](https://github.com
 #### Code review
 
 Your code will be reviewed, in this sequence, by:
-* Travis CI: by running static analysis.<br>
+* Github Actions: by running static analysis.<br>
   If there are problems found, the build will terminate without proceeding to testing.<br>
   Most of the tools will display the cause of the failures in the console;
   if this is not the case, you can run any of the static analysis tools and obtain the reports locally.<br>
   Ensure that the static analysis passes before triggering another build.
-* Travis CI: by building and running tests.<br>
+* Github Actions: by building and running tests.<br>
   If there are failed tests, the build will be marked as a failure.
   You can consult the CI log to find which tests.<br>
   Ensure that all tests pass before triggering another build.
```

Comment:
The line after this:
```
The CI log will also contain the command that will enable running the failed tests locally.
```
is no longer accurate and should be removed.

Reply:
ah correct - removed